django_slack/utils.py

```diff
@@ -10,14 +10,14 @@ class Backend(object):
     def send(self, url, data):
         raise NotImplementedError()
 
-    def validate(self, content_type, content):
+    def validate(self, content_type, content, original_data):
```

Comment:
Makes sense. How about calling it ``original_data`` for now *everywhere* - even though it's not a great name it's easy to ``sed`` later, whilst ``data`` will require actually looking at the code and potentially missing the one in ``SlackException``.